tests/unit/components/TestPools.spec.js

```diff
@@ -0,0 +1,27 @@
+import { render } from "@testing-library/vue";
+import { API } from "aws-amplify";
+
+import TestPools from "@/components/TestPools.vue";
+import { ListPools } from "@/graphql";
+
+jest.mock("aws-amplify", () => ({
+  API: {
+    graphql: jest.fn(),
+  },
+}));
+
+describe("TestPools", () => {
+  beforeEach(() => {
+    API.graphql.mockImplementation((options) => {
+      if (options.query === ListPools) {
+        return Promise.resolve({ data: { listPools: [] } });
+      }
+      throw new Error("Unsupported");
+    });
+  });
+
+  it("should render the headline", () => {
+    const { getByText } = render(TestPools);
+    expect(getByText("🧪 Aktive Test-Pools")).toBeInTheDocument();
```

Comment:
Will this emoji be a problem on different OS? Is it also possible to use the html entity representation (&#x1F9EA;) ?

Reply:
Yup, that was also my concern. But at least on OSX and Ubuntu (in CI) it works 🙂
I couldnt find out how to check using the HTML entity, but i can give it a try again and maybe find a way test for substring 🤷‍♀️